src/test/java/com/wikia/webdriver/common/core/annotations/Driver.java

```diff
@@ -0,0 +1,13 @@
+package com.wikia.webdriver.common.core.annotations;
+
+/**
+ * Created by Ludwik on 2015-02-19. Added LANGUAGE2 Japanese. By P. Archbold on 2015-07-23
+ */
+public class Driver {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)